packages/mui-base/src/utils/useAnchorPositioning.ts

```diff
@@ -205,20 +204,32 @@ export function useAnchorPositioning(
     nodeId,
   });
 
-  // We can assume that element anchors are stable across renders, and thus can be reactive.
-  const reactiveAnchorDep = anchor == null || isElement(anchor);
-  const anchorRef = useLatestRef(anchor);
+  const rerender = useForcedRerendering();
+
```

Comment:
This doesn't seem to be actually used? It's just in the dep array

Reply:
Right, it's a leftover from a previous attempt.